src/main/scala/com/microsoft/hyperspace/index/IndexLogManager.scala

```diff
@@ -44,6 +44,9 @@ trait IndexLogManager {
   /** Returns the latest LogEntry whose state is STABLE */
   def getLatestStableLog(): Option[LogEntry]
 
+  /** Returns Active index log versions */
+  def getActiveIndexVersions(): Seq[Int]
```

Comment:
I noticed that we are using "id" to refer the "version". Should we change it to "id"? WDYT?

Reply:
I think version is more general term to represent it, but the code base is `id`.. might be good to be consistent.
But in the plan, we print it as `LogVersion: ` ?